gateway/requirements.txt

```diff
@@ -18,4 +18,5 @@ drf-yasg>=1.21.7
 cryptography>=41.0.1
 # Django dependency, but we need a newer version (IBMQ#246)
 sqlparse>=0.5.0
-
+qiskit_ibm_runtime
+quantum_serverless
```

Comment:
we do not need that with change above

Reply:
Don't we continue needing `quantum_serverless` for `QiskitObjectsDecoder`? I think right now I would prefer to copy paste this decoder logic due to the problems that we can have between the rename from `quantum_serverless` and `qiskit_serverless`, WDTY?